packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -5217,8 +5225,8 @@ export default class Meeting extends StatelessWebexPlugin {
     this.setupMediaConnectionListeners();
 
     if (this.isMultistream) {
-      this.sendSlotManager.createSlot(mc, MediaType.VideoMain, false);
-      this.sendSlotManager.createSlot(mc, MediaType.AudioMain, false);
+      this.sendSlotManager.createSlot(mc, MediaType.VideoMain, options.audioEnabled);
```

Comment:
when `audioEnabled` is not specified by the client calling `addMedia()` the default value used is `true` (see code inside addMedia), here the type says that it can be undefined and in that case we will set slot to inactive, which is the opposite of what we want. This could be fixed by passing `audioEnabled` instead of `options.audioEnabled` in line 5398, but you should also fix the type that createMediaConnection() takes to reflect the fact that `audioEnabled` cannot be undefined. I think you can do it by using `Required<AddMediaOptions>` but it's also a good practice to define a type that only contains things you really need and use. In this case you're not using most of the things from AddMediaOptions, so we probably should use Pick<> or actually it would be easier to just define a type with the 4 props you actually need (audioEnabled, videoEnabled, etc)

Reply:
We do not need the AddMediaOptions for transcoded meetings. In that case, we cannot mark it as a required parameter, right?

If so, is it okay if I check for those options inside `if(this.isMultiStream)`?

Reply:
As per Webex discussions, used mediaDirections instead of AddMediaOptions.